### PR TITLE
Add highlight for links to guest tutorials and Teachable Machine videos

### PR DESCRIPTION
**Pull request by mikakruschel**
I think it is easy to overlook the links to the guest tutorials and the Teachable Machine videos, so I highlighted "guests" and "Teachable Machine":

image

### CODE REVIEW

File: _Tutorials/index.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ layout: series-index
 
 Here you can find all tutorials made by Daniel Shiffman on TheCodingTrain.
 
-If you are searching for tutorials made by guests, you can check them out [here]({{ site.baseurl }}{% link _GuestTutorials/index.md %}).
+If you are searching for tutorials made by <highlight>guests</highlight>, you can check them out [here]({{ site.baseurl }}{% link _GuestTutorials/index.md %}).
```
Comment (shiffman) [inline _Tutorials/index.md]: For accessibility, we should not use generic terms like "here". People who browse the internet with a screen reader read lists of the links and so meaningful text should always be included. 

For more see: https://www.washington.edu/accessibility/links/ and https://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs

**Comment by shiffman** [comment]
Ooops, I was too slow, if we could correct in a separate pull request that would be great!

File: _Tutorials/index.md (hunk 1 above)
Comment (mikakruschel) [reply _Tutorials/index.md]: The actual links are still called "here", it's only the texts "guests" and "teachable machine" which are highlighted

**Comment by mikakruschel** [comment]
Sorry, my PR title isn't accurate, only some text is highlighted not the actual link.

File: _Tutorials/index.md (hunk 1 above)
Comment (shiffman) [reply _Tutorials/index.md]: Apologies for confusion it's the use of "here" that is a problem. The accessible way to create a link is as follows:

```
If you are searching for tutorials made by <highlight>guests</highlight>, you can check them
out on [the guest tutorial page]({{ site.baseurl }}{% link _GuestTutorials/index.md %})
```

There's probably a shorter way to phrase the above to reduce redundancy too!

Comment (mikakruschel) [reply _Tutorials/index.md]: How about:
```suggestion
For even more tutorials, you can visit the [guest tutorials page]({{ site.baseurl }}{% link _GuestTutorials/index.md %}).
```
And
```suggestion
If you are interested in creating your own machine learning models, have a look at the [Teachable Machine page]({{ site.baseurl }}{% link _TeachableMachine/index.md %}).
```
English is not my native language so I'm open to any critic.

Comment (shiffman) [reply _Tutorials/index.md]: @violetcraze would you mind taking a look at the above discussion and adjusting the language on the site? Thank you! ❤️

Comment (violetcraze) [reply _Tutorials/index.md]: Looks great to me!

**Pull request merged**